src/api/guardian.js

```diff
@@ -1,16 +1,24 @@
 const Guardian = require("guardian-js");
 const guardian_filter = require("./guardian_filter");
+let allArticles = [];
 
-
-const guardianListing = (query, cb) => {
- 
+const guardianListing = (query, page, cb) => {
   const welvonAPI = new Guardian(process.env.GUARDIAN_KEY, false);
   welvonAPI.content
     .search(query, {
-      "show-fields": "headline,trailText,thumbnail,bodyText,page,lastModified"
+      "show-fields": "headline,trailText,thumbnail,bodyText,page,lastModified",
+      page: page
     })
     .then(response => {
-      cb(JSON.parse(response.body.response));
+      json_res = JSON.parse(response.body).response;
+      const { currentPage, pageSize } = json_res;
```

Comment:
why not just do `const { currentPage, pageSize } = JSON.parse(response.body)`

Reply:
I need to use json_res as well